src/components/modal/Modal.scss

```diff
@@ -144,25 +146,50 @@
             padding: 0;
         }
 
+        .modal-header-container {
+            position: absolute;
+            top: 0;
+            left: 0;
+            display: flex;
+            align-items: center;
+            width: 100%;
+            height: $responsive-modal-header-height;
+            background-color: $white;
```

Comment:
With the added `margin-top` for `modal-content`, I don't think we need the background-color anymore since there shouldn't be anything under the header.

Reply:
You're right, it's not needed anymore. I've removed it.